.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+## What type of PR is this? (check all applicable)
+
+- [ ] Feature
+- [ ] Bug Fix
+- [ ] Refactor
+- [ ] Optimization
+- [ ] Test
+- [ ] Build
+- [ ] Documentation Update
+
+## Description
+
+## QA Instructions, Screenshots, Recordings
+
+_Please replace this line with instructions on how to test your changes, a note
+on the devices and browsers this has been tested on, as well as any relevant
+images for UI changes._
+
+### UI accessibility checklist
+_If your PR includes UI changes, please utilize this checklist:_
+- [ ] Semantic HTML implemented?
+- [ ] Keyboard operability supported?
+- [ ] Checked with [axe DevTools](https://www.deque.com/axe/) and addressed `Critical` and `Serious` issues?
+- [ ] Color contrast tested?
```

Comment:
Great job using resources, but please take a look at the code before copying and pasting! Turn this into your own.